Profiles/cio-Symptom.xml

```diff
@@ -0,0 +1,153 @@
+      <sliceName value="relationCondition" />
+      <min value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-RelationCondition" />
+      </type>
+      <isModifier value="false" />
+    </element>
+    <element id="Observation.identifier">
+      <path value="Observation.identifier" />
+      <short value="IdentificationNumber" />
+      <definition value="Globally unique number that identifies the instantiation of the information model. The number is composed of an identification of the issuer organization and a unique number assigned by this organization." />
+      <alias value="Identificatienummer" />
+      <mapping>
+        <identity value="cio-dataset-200-beta2-20231214" />
+        <map value="cio-dataelement-20-780" />
+        <comment value="IdentificationNumber" />
+      </mapping>
+    </element>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <short value="SymptomName" />
+      <definition value="Name and code of the symptom via selection from a code list. Represents the symptom which a condition in the patient presents itself." />
+      <alias value="SymptoomNaame" />
+      <binding>
+        <strength value="required" />
+        <description value="Identification of the symptom." />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.26.11.35--20240501114333" />
+      </binding>
+      <mapping>
+        <identity value="cio-dataset-200-beta2-20231214" />
+        <map value="cio-dataelement-20-901" />
+        <comment value="SymptomName" />
+      </mapping>
+    </element>
+    <element id="Observation.code.text">
+      <path value="Observation.code.text" />
+      <short value="FurtherSpecificationSymptomName" />
+      <definition value="More detailed specification of the symptomName that has been selected from the code list, because that list does not (yet) contain the required level of detail." />
+      <alias value="NadereSpecificatieSymptoomNaam" />
+      <mapping>
+        <identity value="cio-dataset-200-beta2-20231214" />
+        <map value="cio-dataelement-20-902" />
+        <comment value="FurtherSpecificationSymptomName" />
+      </mapping>
+    </element>
+    <element id="Observation.subject">
+      <path value="Observation.subject" />
+      <min value="1" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Patient" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Group" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Device" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Location" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient" />
+      </type>
+    </element>
+    <element id="Observation.interpretation">
```

Comment:
I don't think this mapping is correct. The definition of `Observation.interpretation` is:

_A categorical assessment of an observation value. For example, high, low, normal._

In this profile there currently is no value (the value could be something like 'Present' or 'Absent', see e.g. http://hl7.org/fhir/R4/observation-example-vomiting.xml.html), so it doesn't make sense to make an interpretation on that. Perhaps `Observation.component' is better suited to map the SymptomSeverity concept on.

Reply:
Could you check if other zibs have similar Severity concepts, and how these are mapped in FHIR?

Reply:
Ah right, .interpretation is not a correct mapping then. Unfortunately there are no Severity concepts that map to something in an Observation that I could take inspiration from. Most of the time there is a dedicated `.severity` element. Such as in Condition. However, observation.component is not either imo. Definition of `Observation.component` is:

_Some observations have multiple component observations. These component observations are expressed as separate code value pairs that share the same attributes. Examples include systolic and diastolic component observations for blood pressure measurement and multiple component observations for genetics observations._

Maybe an extension? Or using a Condition for Symptom in the end?

Reply:
Additional food for thought; for the nl-core profiles we had a discussion on `.component` versus `.hasMember`, and there we use the following guideline:

Only use `Observation.component` for any supporting functional zib element that cannot reasonably be interpreted and used outside the scope of the Observation it is a component of. Otherwise, a separate profile on Observation is created for this functional element, which is referenced in `Observation.hasMember` of the 'main' profile. Looking at the DefinitionCode associated with a concept can help identifying this. If the code is a descendant of SNOMED 404684003 (Clinical finding) this could indicate a separate Observation.

For instance, BodyWeight.Clothing or O2Saturation.ExtraOxygenAdministration are mapped on `.component`. These concepts give additional information about the measurement of the main value and can therefore not be considered as separate Observations. Since the severity doesn't make much sense as a separate Observation as well, I still think it'd be justified to map it on `.component`. For Symptom there is not really a `.value`, however the 'measurement' in this case is the acknowledgement of there being a Symptom (which is indicated by `.code`). Again, it would be nice to discuss this with Pieter to see what the latest insights are from an nl-core point of view.

Reply:
Decision to map element to Observation.component. Based on : http://hl7.org/fhir/R5/observation.html#gr-comp